### ISSUE TITLE
Switching to nodejs 

### ISSUE THREAD

**Issue by LegolasGChief** [ArqTras/nodejs-pool#11]
Hey I am trying to switch my pool to nodejs from universal.
https://github.com/ArqTras/nodejs-pool/wiki/Leaf-Node-Wiki

Is this a full tutorial for setting it up? Can you give any advise?
I have setup a virtual machine using VirtualBox and Ubuntu server 16.04 to test setting up the pool before I do it on my main server.

I have Aeon compiled and have synced aeond. I did a git clone of pool software.

What do I need to do now? The tutorial is a little hard to follow. I tried using the aeon development script from snipa22 deployment folder but doesnt seem to work correctly.

Any idea where I go from here or can link to help? 

Have been wanting to do this for a while but I am a little stuck here what to do

Thanks :)

**Comment by ArqTras** [comment]
To setup aeon you should use this wiki:
https://github.com/ArqTras/nodejs-pool/wiki
Only change branch for aeon when you git clone
Also I run manually aeond without running it as service (dont have time to correct that)
and simple wallet all on screen

Try to install it using that bash script
https://github.com/ArqTras/nodejs-pool/blob/aeon/deployment/deploy_aeon.bash

**Comment by LegolasGChief** [comment]
Hey I got it working :) thanks a lot

Can close this
